ros/src/computing/perception/detection/packages/cv_tracker/CMakeLists.txt

```diff
@@ -178,6 +178,17 @@ ADD_DEPENDENCIES(obj_reproj
   calibration_camera_lidar_generate_messages_cpp
 )
 
+# check whether jsk library is available or not
+find_package(jsk_recognition_msgs)
+if (jsk_recognition_msgs_FOUND)
+  # if jsk plugin is found, activate macro
+  SET_TARGET_PROPERTIES(obj_reproj
+    PROPERTIES COMPILE_FLAGS
+    "-DIS_FOUND_JSK_PLUGIN")
```

Comment:
I suppose `USE_JSK_PLUGIN` or `HAVE_JSK_PLUGIN` is better name rather than `IS_FOUND_JSK_PLUGIN`. `IS_FOUND_JSK_PLUGIN` looks like predicate function.